doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/configuring-EAP-security-management-provider-CLI-mode-proc.adoc

```diff
@@ -0,0 +1,52 @@
+[id='configuring-EAP-security-management-provider-CLI-mode-proc_{context}']
+
+= Configuring the {EAP} security management provider based on property files and CLI mode
+
+To use the {EAP} security management provider based on property files and CLI mode, complete the following steps:
```

Comment:
If the steps immediately followed this statement, the : would be correct. I suggest changing to:
...complete the steps in this procedure.

Reply:
Kalyani, I don't see this change in the rendered draft, or here.